packages/apputils-extension/src/index.ts

```diff
@@ -284,6 +283,28 @@ const print: JupyterFrontEndPlugin<void> = {
   }
 };
 
+/**
+ * Update the browser title based on the workspace and the current
+ * active item.
+ */
+async function updateTabTitle(
+  workspace: string,
+  db: IStateDB,
+  trans: TranslationBundle
+) {
+  const data: any = await db.toJSON();
+  // Number of restorable items, minus the layout restorer data
+  const count: number = Object.keys(data).length - 1;
+  let current: string = data['layout-restorer:data']?.main?.current;
+  current = current.split(':')[1];
+
+  if (workspace.startsWith('auto-')) {
+    document.title = `${current}(${workspace}:${count}) - JupyterLab`;
```

Comment:
Thanks for working on this @jess-x, maybe adding a space between the name and the parens would make it more clear :) ?

Reply:
I agree, adding a space in between does look better. I'd make the change